PHP/PHP Source.sublime-syntax

```diff
@@ -128,7 +128,32 @@ contexts:
           pop: true
         - match: (?=\S)
           pop: true
-    - match: \b(break|c(ase|ontinue)|d(e(clare|fault)|ie|o)|e(lse(if)?|nd(declare|for(each)?|if|switch|while)|xit)|for(each)?|if|return|switch|use|while)\b
+    - match: |-
+        (?x)\b(?:
```

Comment:
You could easily use `\b(?x:` to simplify the term a bit. It also applies to most of the following hunks.